app/jinja_filters.py

```diff
@@ -709,8 +709,11 @@ def map_list_collector_config(
             else None
         )
 
+        iconVisuallyHiddenText = "Section complete" if icon == "check" else None
```

Comment:
Where did the wording "Section complete" come from for this? Feels a little bit confusing to me given you are already within the context of a section, and section is a term that may be referenced elsewhere

Reply:
The wording was just from the ticket itself

Reply:
As discussed, I updated the text to match the DAC report and will mention this in refinement as well as the ambiguity of what is "complete or not"